src/sections/Resources/Resources-grid/filters.js

```diff
@@ -67,7 +67,6 @@ const Navigation = (props) => {
   const technologies = counting.technology.group;
   const meshes = counting.mesh.group;
 
-
```

Comment:
Revert this file change, space...

Reply:
This is not visible on my machine.

Reply:
This had been sorted too. The code is well-formatted with a single space instead of a double.

Reply:
> The code automatically gets formatted when you make a commit by a pre-commit linter. You just need to make changes in the first file not other

This is adjusted now.